AdminWebsite/AdminWebsite.AcceptanceTests/Data/AssertConference.cs

```diff
@@ -40,10 +40,16 @@ private static void AssertEndpoints(IReadOnlyCollection<EndpointResponse2> heari
         {
             foreach (var conferenceEndpoint in conferenceEndpoints)
             {
-                var hearingEndpoint = hearingEndpoints.First(x => x.Id.Equals(conferenceEndpoint.Id));
+                var hearingEndpoint = hearingEndpoints.First(x => x.Sip.Equals(conferenceEndpoint.Sip_address));
+ 
+                if (conferenceEndpoint.Defence_advocate == null)
+                {
+                    hearingEndpoint.Defence_advocate_id.Should().BeNull();
```

Comment:
would it not be best to assert the defence advocate id is that of a participant in the same conference?

Reply:
What is public string Defence_advocate { get; set; } on the hearing endpoint response? Is it display name?

Reply:
I believe so. may have to double-check the contract or check the mapper

Reply:
looks like it's the username

Reply:
@shaed-parkar Updated!